lib/data_hygiene/anonymise_email_addresses.sql

```diff
@@ -21,8 +21,9 @@ WHERE created_at < current_timestamp - interval '1 day';
 CREATE TABLE addresses (id SERIAL, address VARCHAR NOT NULL);
 
 # Copy all email addresses into the table.
+# Ignore nulled out subscriber addresses.
 INSERT INTO addresses (address)
-  SELECT address FROM subscribers
+  SELECT address FROM subscribers WHERE address IS NOT NULL
```

Comment:
Will this mean that for subscribers that are nullified, we will see `Email`s with non-anonymised addresses?

Reply:
I don't think so - we aren't joining on `subscribers`  when we anonymise `addresses` in the `emails` table.


```
UNION DISTINCT
SELECT address FROM emails
```

will take care of any addresses in `emails` table.

Reply:
Ah yes, good point!